src/outputs/outputs_main.py

```diff
@@ -8,14 +8,21 @@
 import src.outputs.short_form_out as short
 import src.outputs.map_output_cols as map_o
 
+from src.outputs.short_form import run_short_form
+from src.outputs.tau import run_tau
+
+
 OutputMainLogger = logging.getLogger(__name__)
 
 # Get the shortform schema
 short_form_schema = toml.load("src/outputs/output_schemas/frozen_shortform_schema.toml")
 
+# Get the Tau Argus output schema
+tau_schema = toml.load("src/outputs/output_schemas/tau_schema.toml")
```

Comment:
Should be done inside the function maybe?

Reply:
Agree, it should be read inside the function from the config

Reply:
I don't think this has been actioned yet

Reply:
Removed loading schemas in outputs_main.py. They are called from within corresponding modules. Also removed import toml